diff -rup lohit-telugu-2.4.5/Lohit-Telugu.sfd lohit-telugu-2.4.5_mod/Lohit-Telugu.sfd
--- lohit-telugu-2.4.5/Lohit-Telugu.sfd	2011-07-06 14:30:43.139112937 +0530
+++ lohit-telugu-2.4.5_mod/Lohit-Telugu.sfd	2011-07-06 14:30:26.151108304 +0530
@@ -20,7 +20,7 @@ OS2Version: 1
 OS2_WeightWidthSlopeOnly: 0
 OS2_UseTypoMetrics: 1
 CreationTime: 1153280036
-ModificationTime: 1281370618
+ModificationTime: 1309942824
 PfmFamily: 17
 TTFWeight: 400
 TTFWidth: 5
@@ -2883,15 +2883,14 @@ EndShort
 LangName: 1033 "" "" "Regular" "FontForge 1.0 : Lohit Telugu : 19-7-2006" "" "Version 2.4.5" 
 GaspTable: 1 65535 2
 Encoding: UnicodeBmp
-Compacted: 1
 UnicodeInterp: none
 NameList: Adobe Glyph List
 DisplaySize: -48
 AntiAlias: 1
 FitToEm: 1
-WinInfo: 0 28 12
+WinInfo: 8344 28 11
 AnchorClass2: "Anchor-1"  "'blwm' Below Base Mark in Telugu lookup 0-1" "Anchor-0"  "'blwm' Below Base Mark in Telugu lookup 0" 
-BeginChars: 65989 553
+BeginChars: 65989 554
 
 StartChar: .notdef
 Encoding: 65536 -1 0
@@ -5378,7 +5377,7 @@ Encoding: 3114 3114 41
 Width: 572
 VWidth: 819
 GlyphClass: 2
-Flags: HW
+Flags: W
 AnchorPoint: "Anchor-1" 286 0 basechar 0
 LayerCount: 2
 Fore
@@ -5789,7 +5788,7 @@ Encoding: 3120 3120 47
 Width: 540
 VWidth: 819
 GlyphClass: 2
-Flags: HW
+Flags: W
 AnchorPoint: "Anchor-1" 270 0 basechar 0
 LayerCount: 2
 Fore
@@ -38564,7 +38563,7 @@ Encoding: 65891 -1 440
 Width: 0
 VWidth: 819
 GlyphClass: 4
-Flags: HW
+Flags: W
 AnchorPoint: "Anchor-1" -300 0 mark 0
 LayerCount: 2
 Fore
@@ -38790,7 +38789,7 @@ Encoding: 65894 -1 443
 Width: 0
 VWidth: 819
 GlyphClass: 4
-Flags: HW
+Flags: W
 AnchorPoint: "Anchor-1" -321 0 mark 0
 LayerCount: 2
 Fore
@@ -45780,5 +45779,37 @@ SplineSet
  411.018 238.665 411.018 238.665 425 289 c 0,0,1
 EndSplineSet
 EndChar
+
+StartChar: uni20B9
+Encoding: 8377 8377 553
+Width: 622
+Flags: HW
+LayerCount: 2
+Fore
+SplineSet
+358 499 m 0,0,1
+ 359 508 359 508 359 513 c 1,2,-1
+ 59 513 l 1,3,-1
+ 85 576 l 1,4,-1
+ 346 576 l 1,5,6
+ 340 588 340 588 331 598 c 0,7,8
+ 280 653 280 653 165 653 c 2,9,-1
+ 59 653 l 1,10,-1
+ 85 716 l 1,11,-1
+ 583 716 l 1,12,-1
+ 557 653 l 1,13,-1
+ 367 653 l 1,14,15
+ 406 630 406 630 420 576 c 1,16,-1
+ 583 576 l 1,17,-1
+ 557 513 l 1,18,-1
+ 429 513 l 1,19,20
+ 429 357.613 429 357.613 171 339 c 1,21,-1
+ 507 -1 l 1,22,-1
+ 407 -1 l 1,23,-1
+ 70 336 l 1,24,-1
+ 70 395 l 1,25,26
+ 345.891 393 345.891 393 358 499 c 0,0,1
+EndSplineSet
+EndChar
 EndChars
 EndSplineFont
